avocado/plugins/assets.py

```diff
@@ -203,6 +215,32 @@ def fetch_assets(test_file):
     return success, fail
 
 
+class FetchAssetJob(JobPreTests):  # pylint: disable=R0903
+    """
+    Implements the assets fetch job pre tests. This has the same effect of
+    running the 'avocado assets fetch INSTRUMENTED', but it runs during the
+    test execution, before the actual test starts.
+    """
+    name = "fetchasset"
+    description = "Fetch assets before the test run"
+
+    def __init__(self, config=None):
+        pass
```

Comment:
`config` is been passed but not used. Is that expected?

Reply:
This is a workaround as `avocado/core/extension_manager.py`, line 63, `obj = plugin(**invoke_kwds)`, calls all the plugins with a parameter. As JobPreTests class do not define any, we need to define at least one to be able to run it, or we will have an error message like this:

```
Failed to load plugin from module "avocado.plugins.assets": TypeError('FetchAssetJob() takes no arguments') :
  File "~/src/avocado/avocado.dev/avocado/core/extension_manager.py", line 63, in __init__
    obj = plugin(**invoke_kwds)
```

Reply:
Ok, let's move on. :)